cuda/src/main/java/org/bytedeco/cuda/presets/nvcomp.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 Samuel Audet
```

Comment:
We could put the name of your lab here I guess?

Reply:
Changed